{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly.
+
+### PR Cover Letter
+- [ ] Include A detailed description of the changes in the body of the PR
```

Comment:
I'm a little confused by check boxes that are there for the developer to mark off before they save the updated cover letter. My preference would be to only have check boxes that should remain for the reviewers to see. Maybe we need another convention for content that should be replaced. Maybe all caps?

    REPLACE THIS WITH A DETAILED DESCRIPTION OF YOUR CHANGES